borg/helpers.py

```diff
@@ -312,17 +312,50 @@ def load_excludes(fh):
     both line ends are ignored.
     """
     patterns = (line for line in (i.strip() for i in fh) if not line.startswith('#'))
-    return [parse_pattern(pattern) for pattern in patterns if pattern]
+    return [parse_exclude_pattern(pattern)
+            for pattern in patterns if pattern]
```

Comment:
oh, i did not even see you changed the function name. X-)
what i asked about was the formatting.
and if you would not change that, the real change would be more obvious.